starknet_py/net/rpc_schemas/rpc_schemas.py

```diff
@@ -304,3 +314,15 @@ class DeployTransactionResponseSchema(SentTransactionSchema):
     @post_load
     def make_dataclass(self, data, **kwargs):
         return DeployTransactionResponse(**data)
+
+
+class PendingTransactionsSchema(Schema):
```

Comment:
Are we testing these? Perhaps if devnet doesn't support pending transactions, we should mock one based off the RPC response and add tests for these.